src/js/common/schemaform/state/reducers.js

```diff
@@ -12,6 +15,23 @@ import {
 } from '../state/helpers';
 
 export default {
+  [OPEN_REVIEW_CHAPTER]: (state, action) => {
+    const openChapters = _.concat(
```

Comment:
Decided to go with an array for now. If we opt to isolate the accordion behavior, this could be changed into an object that stores more stuff (e.g. errorState, openState, etc).